### DISCUSSION TITLE
How can I delete entries from jumplist?

### DISCUSSION THREAD

**Discussion by sanket143-toddleapp** [helix-editor/helix#7319] [Q&A] [answered]
After a while, my jumplist ends up being a huge mess, How can I keep it organized? perhaps by opening jump list of only my current file or ability to remove jumplist pointers?

**Comment by gabydd** [comment, marked as answer]
It's not currently possible to clear any entries of the jumplist, that's tracked by: https://github.com/helix-editor/helix/issues/4757

**Comment by ArcSpecter** [comment]
this is kinda a major issue, shouldn't it be elevated? because its a big hassle. Editor is not quite complete without that feature. its basically as deleting marks in my vim

**Reply by kirawi** [reply]
It's blocked on some discussion/changes. See https://github.com/helix-editor/helix/pull/4522
